### ISSUE TITLE
Please support the whitelist (allow detection of accessibility)

### ISSUE THREAD

**Issue by Marukon** [alexdonh/no-accessibility-services#2]
This is a great module that addresses the pain point of some apps requiring accessibility detection. However, there is an issue where my ad-blocking app (based on accessibility principles) cannot detect whether its own accessibility is enabled, leading to functionality issues. Therefore, I am requesting the addition of a whitelist feature that allows apps on the list to detect accessibility. Thank you.

**Comment by alexdonh** [comment]
> This is a great module that addresses the pain point of some apps requiring accessibility detection. However, there is an issue where my ad-blocking app (based on accessibility principles) cannot detect whether its own accessibility is enabled, leading to functionality issues. Therefore, I am requesting the addition of a whitelist feature that allows apps on the list to detect accessibility. Thank you.

Untick system framework, make sure you tick only the apps you want to hide accessibility services. Restart. 

Sorry I am just too lazy to bump an update just to untick System framework by default (as this is actually not needed).

**Comment by Marukon** [comment]
Thank you for your reply. I will untick system framework.

**Comment by hung319** [comment]
> > This is a great module that addresses the pain point of some apps requiring accessibility detection. However, there is an issue where my ad-blocking app (based on accessibility principles) cannot detect whether its own accessibility is enabled, leading to functionality issues. Therefore, I am requesting the addition of a whitelist feature that allows apps on the list to detect accessibility. Thank you.
> 
> Untick system framework, make sure you tick only the apps you want to hide accessibility services. Restart.
> 
> Sorry I am just too lazy to bump an update just to untick System framework by default (as this is actually not needed).

Hello, if I don't check the system framework box, the app I hide will crash and I won't be able to access it. Is there any way to fix this?
